### ISSUE TITLE
Problem rendering list with a single item

### ISSUE THREAD

**Issue by hybridvision** [deanblackborough/php-quill-renderer#132]
Hello @deanblackborough,

Thank you very much for all your work on this library! 

I came across a problem when rendering a list that only has a single item and is preceeded by some other content (even a single line break). In this case, the list tag is never closed and it results in invalid HTML.

**Example delta:**

```json
{
  "ops": [
    {
      "insert": "Normal paragraph\n\nList with single item"
    },
    {
      "attributes": {
        "list": "bullet"
      },
      "insert": "\n"
    },
    {
      "insert": "\nAnother normal paragraph\n"
    }
  ]
}
```

**Output:**
```html
<p>Normal paragraph </p>
<ul>
    <li>List with single item</li>
    <p>
        <br /> Another normal paragraph 
        <br />
    </p>
```

**Expected Output:**
```html
<p>Normal paragraph </p>
<ul>
    <li>List with single item</li>
</ul>
<p>
    <br /> Another normal paragraph 
    <br />
</p>
```

If there's nothing _before_ the list, it works correctly. Likewise, the list is closed properly when there's more than one list item.

I tested this on both v3.18.1 and v4.00.0 of your library inside my app but the result was the same. On the client side, I'm using Quill v1.3.7. I also observed the same behaviour on [your site](https://www.transmute-coffee.com/php-quill-renderer.php#demo).

Do you think this can be fixed? Appreciate your time!

**Comment by hybridvision** [comment]
Hi Dean,

Wow, that was quick! Many thanks for your response and please let me know if there's anything I can assist with in terms of testing :)

**Comment by deanblackborough** [comment]
@hybridvision 

This has been fixed in #133 and I've released it as v4.00.2. I wasn't closing single item lists under specific conditions; lists are now always closed. 

I've tested with your delta, and it produces valid HTML.

Thank you for taking the time to report the bug and for providing as much information as you did, it made it easy to identify the problem area.

**Comment by hybridvision** [comment]
Thank you Dean, I really wasn't expecting such a speedy resolution!

I confirm that it works perfectly now for me too :)
